test/config/destination-policy.yaml

```diff
@@ -0,0 +1,10 @@
+# Disable mTLS for connections to the API server
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
```

Comment:
where do we apply?
do we need to system namespace?

Reply:
it's applied in both istio-system and knative-eventing as part of install-dependencies.sh